tutorials/custom-renderer.html

```diff
@@ -128,7 +128,7 @@ <h3>
                 <p>
                     And Thats it! You created a Renderer and A Render Batch.
                     (Go You! You da best! :D)
-                    If you have any questions, ping @VoxelRifts#3089 in the Games-With-Gabe discord.
+                    If you have any questions, ping anybody from Helper role in the Games-With-Gabe discord.
```

Comment:
This is set to voxelrifts because he is the author of the batch rendering API, he understands it best.